scripts/actions/mobskills/questionmarks_needles.lua

```diff
@@ -9,15 +11,24 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    -- from http://ffxiclopedia.wikia.com/wiki/%3F%3F%3F_Needles
-    -- "Seen totals ranging from 15, 000 to 55, 000 needles."
-    local needles = math.random(15000, 55000) / skill:getTotalTargets()
+        -- from http://ffxiclopedia.wikia.com/wiki/%3F%3F%3F_Needles
+        -- "Seen totals ranging from 15, 000 to 55, 000 needles."
+    if mob:getID() == zones[xi.zone.ABYSSEA_ALTEPA].mob.CUIJATENDER then
```

Comment:
our normal pattern is to `mob:getPool()` but I think I like this better thanks to the magic of no longer being forced to use magic number inside IDs.lua 👍

Reply:
in fact I think I'd be keen on approving PRs converting some older scripts to this method